### ISSUE TITLE
Is perldoc.perl.org abandoned?

### ISSUE THREAD

**Issue by tm604** [OpusVL/perldoc.perl.org#114]
Not seeing any official responses on the bugtracker over the last few months, and at the time of writing the perldoc.perl.org page is showing an outdated (5.30.0) version of Perl.

Has this been abandoned? If so, what would be the steps for locating an alternative maintainer + transferring ownership to them?

As a site under the official `perl.org` domain, it looks pretty bad to have so many [unanswered questions and bugs](https://github.com/OpusVL/perldoc.perl.org/issues) and even trivial one-character [PRs left open](https://github.com/OpusVL/perldoc.perl.org-engine/pull/11). I've been recommending https://perldoc.pl/ instead, but I think the key requirement is that - no matter which system/owner/layout/font the site has - it **needs to be kept up to date**, otherwise it just looks like Perl itself is no longer supported...

If it's a lack of time, a call for volunteers would seem to be appropriate?

**Comment by haukex** [comment]
Bump

**Comment by hakonhagland** [comment]
If the repo is abandoned, it could still be forked and new maintainers assigned to the fork? What do you think? What is the best approach?

**Comment by haukex** [comment]
I'd nominate @Grinnz's perldoc.pl (<https://github.com/Grinnz/perldoc-browser>) to replace the current perldoc.perl.org.

**Comment by hakonhagland** [comment]
@haukex Yes that seems to be a solution. Then, any web requests to https://perldoc.perl.org/  should be redirected to https://perldoc.pl/ as soon as possilble. As it is now, perldoc.pl does not show up when you google for instance `perlre` or `perlop`. Any newcomer to Perl can believe the documentation is not good or badly maintained. By the way, who owns the domain https://perldoc.perl.org/  ?

**Comment by haukex** [comment]
I'm not sure, but I'd guess TPF. (It's not listed on https://noc.perl.org/)

**Comment by hakonhagland** [comment]
@haukex I sent a mail to the maintainer Paul Webster listed at https://perldoc.perl.org/about.html offering to help with maintenance of the site. The mail was sent May 5th and I have still not got a reply, wonder what is going on?

**Comment by choroba** [comment]
The site seems to display the latest version now. The issue can be closed.
